### PR TITLE
[SL-8504] Event dispatcher

### PR DESCRIPTION
**Pull request by darkcl**

### CODE REVIEW

**Comment by darkcl** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://shopline.atlassian.net/browse/SL-8504">SL-8504</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 86fd343bf9798d15b5107dc3b029df47b64aef17
</p>

**Comment by darkcl** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://shopline.atlassian.net/browse/SL-8504">SL-8504</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 86fd343bf9798d15b5107dc3b029df47b64aef17
</p>

**Comment by darkcl** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://shopline.atlassian.net/browse/SL-8504">SL-8504</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 86fd343bf9798d15b5107dc3b029df47b64aef17
</p>

File: internal/senders/senders.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+// SendEvent - #HttpEventSender SendEvent
+func (c *HTTPEventSender) SendEvent(e models.IncomingEvent, config models.Configuration) error {
+	url := config.CallbackURL
+	payload, err := json.Marshal(e)
+	if err != nil {
+		return err
+	}
+
+	req, reqErr := http.NewRequest("POST", url, bytes.NewBuffer(payload))
```
Comment (Seitk) [inline internal/senders/senders.go]: probably can split another file for request formatter later on, adding request method or even the way of callback

**Pull request merged**